Dockerfile

```diff
@@ -0,0 +1,61 @@
+# translate/pootle
+#
+# VERSION	0.0.1
+
+FROM debian
+
+MAINTAINER Ryan Northey <ryan@synca.io>
+
+RUN apt-get update && \
+    apt-get install -y \
+            python-pip \
+	    libxml2-dev \
+	    libxslt-dev \
+	    python-dev \
+	    zlib1g-dev \
+	    build-essential \
+	    redis-server \
+	    sudo && \
+    apt-get clean && \
+    pip install virtualenv && \
+    groupadd -r pootle && \
+    useradd -m -d /home/pootle \
+            -k /etc/skel \
+            -s /bin/bash \
+            -g pootle pootle && \
+    /etc/init.d/redis-server start && \
+    sudo -u pootle bash -c "\
+    	 mkdir ~/dev/env -p && \
+	 cd ~/dev/env && \
+    	 virtualenv . && \
+    	 . bin/activate && \
+    	 pip install --upgrade pip && \
+    	 pip install pootle && \
+	 pootle init && \
+    	 echo 'DEBUG=True' >> ~/.pootle/pootle.conf && \
+    	 pootle migrate --noinput && \
+    	 pootle initdb && \
+	 echo \"from django.contrib.auth import get_user_model; \	       
+	        from django.conf import settings; \
+	       	from django.core import management; \
+	       	from allauth.account.models import EmailAddress; \
+	       	user = get_user_model().objects.create_superuser('admin', 'admin@example.com', 'pootle'); \
+	       	EmailAddress.objects.create(user=user, email='admin@example.com', primary=True, verified=True) \" \
+	    | pootle shell"
```

Comment:
i dont believe so:
http://stackoverflow.com/questions/6244382/how-to-automate-createsuperuser-on-django

also even if you can - you still need to auto verify the email address to make it ready for login

```from django.conf import settings;``` 

does need to be removed though